portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/RemoteAppsRepeatableFields.testcase

```diff
@@ -0,0 +1,72 @@
+definition {
+
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Remote Apps";
+	property testray.main.component.name = "Remote Apps";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			RemoteApps.tearDown();
+
+			Navigator.gotoPage(pageName = "Home");
+
+			while (IsElementPresent(locator1 = "Navigation#ANY_SITE_NAV_BUT_HOME")) {
+				var layoutName = selenium.getText("Navigation#ANY_SITE_NAV_BUT_HOME");
+
+				JSONLayout.deletePublicLayout(
+					groupName = "Guest",
+					layoutName = "${layoutName}");
+
+				Navigator.gotoPage(pageName = "Home");
+
+				PortalInstances.tearDownCP();
+			}
+		}
+	}
+
+	@description = "This is a test for LPS-164433. The JSN URL can be repeatably clicked."
+	@priority = "3"
+	test JSURLCanBeRepeated {
+		property portal.acceptance = "true";
+
+		task ("Given Remote Application > Custom Elements create page") {
+			JSONLayout.publishLayout(
+				groupName = "Guest",
+				layoutName = "Test Page");
+
+			RemoteApps.goToRemoteAppsPortlet();
+
+			RemoteApps.addCustomElement(
+				entryHtmlName = "${customElementHtmlName}",
+				entryName = "${customElementName}",
+				entryURL = "${customElementURL}");
+		}
+
+		task ("When click repeatable button for JS URL") {
+			RemoteAppsEntry.addCSSURLRow();
+
+			RemoteAppsEntry.addCSSURLRow();
```

Comment:
addCSSURLRow is not the same as JS URL. We're looking for `RemoteAppsEntry.addURLRow();` also, we're only supposed to click 1 time. Multiple times should be covered in different testcase https://issues.liferay.com/browse/LPS-164435